src/components/gallery/HdGalleryCarousel.vue

```diff
@@ -105,6 +109,10 @@ export default {
         paddingTop: `${100 / this.aspectRatio}%`,
       };
     },
+    usePictureElement() {
+      const currentItem = this.items[this.currentIndex];
+      return currentItem.thumbnailPictureSources && currentItem.thumbnailPictureSources !== {};
```

Comment:
Are you sure about this check?

Reply:
You can try it: 

```javascript
var a = {}; 
console.log(a === {}); // false
```

For checking if object has no keys, this is not a good check. You might want to use something like this instead 

```javascript
Object.entries(obj).length === 0
```

or

```javascript
Object.keys(obj).length === 0
```

and if you want to be super sure, you can additionally check the constructor - but above is probably already :+1:

Reply:
👍